README.md

```diff
@@ -29,7 +33,30 @@ fullUrl.searchParams.append('foo', '123');
 fullUrl.searchParams.append('bar', '456');
 fullUrl.hash = 'heading-1';
 
-fullUrl.toString(); // 'http://www.google.com/a/b/cd?foo=123&bar=456#heading-1'
+console.log(fullUrl.toString()); // 'http://www.google.com/a/b/cd?foo=123&bar=456#heading-1'
+```
+
+### Joining Path Parts
```

Comment:
Let's call this 'paths' rather than 'parts', as it is more specific to this use-case.
```suggestion
### Joining paths
```